portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,29 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ana-Maria's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Hi! I am Ana-Maria and this is my about me page</h1>
+      <p>I am intern for Google's STEP program this year and I'm excited
+          to learn new things. I am 21 years old and I study Computer Science
+          in Bucharest, Romania.</p>
+
+      <p>Here you can see me:</p>
+      <img src="images/me.jpeg" width="600"/>
```

Comment:
Here and elsewhere:

Do not close void elements:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes